charts/mlrun-ce/templates/_helpers.tpl

```diff
@@ -226,3 +226,14 @@ Model monitoring DSN
 {{- end -}}
 {{- end -}}
 
+{{/*
+serviceAccounts for AWS
+*/}}
+{{- define "aws.serviceAccounts" -}}
+mlrun-api-aws-sa
+mlrun-jobs-aws-sa
+ml-pipeline-ui-sa
+ml-pipeline-sa
+argo-sa
+{{- end -}}
```

Comment:
where is this being used?

Reply:
Probably nowhere, just legacy from CFN era but I don't want to delete yet because no one knows for sure. Marked for deletion